### PR TITLE
DS-work-book: creating and updating for session 26/8

### PR DESCRIPTION
**Pull request by SonJoseDev**

### CODE REVIEW

File: work-book [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Session 26/8
+	- created repo in github and maneged access 
+	- cloned the 6.824 mit repo
+	- installed golang 
+	- built and run the sample project of lab 01 map reduce
```
Comment (samabraha) [inline work-book]: 'run' should be 'ran'

File: work-book [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+	- agreed to read the https://pdos.csail.mit.edu/6.824/labs/guidance.html
+	- agreed to start the go course https://learning.oreilly.com/library/view/learning-go/9781492077206/
```
Comment (samabraha) [inline work-book]: Url should be replaced with https://learning.oreilly.com/videos/ultimate-go-programming/9780135261651/

File: work-book (hunk 1 above)
Comment (SonJoseDev) [reply work-book]: Done

File: work-book (hunk 2 above)
Comment (SonJoseDev) [reply work-book]: Done

**Review by samabraha — APPROVED** [review]
Kudos!

**Pull request merged**